Dockerfile

```diff
@@ -4,11 +4,13 @@ LABEL description="Sentry Issues & Events Exporter"
 
 WORKDIR /app
 
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
creating a pip cache in docker is counter-productive in docker because the cache will never be reused but it will take space on the image, for no purpose